### DISCUSSION TITLE
Loss of cognition and personalized music

### DISCUSSION THREAD

**Discussion by john-** [jcorporation/myMPD#428] [General]
I watched a documentary on how music can be beneficial to those that are in cognitive decline.  Examples shown were those people that had Alzheimer's Disease that would physically/mentally transform when listening to music they enjoyed before the degradation of their cognition.   I recommend anyone interested in this concept to look into it so will not attempt to explain it.

Not to get too dark but it got me thinking about how I can prepare for something like that for myself and others.

Can something like myMPD be used as a  way of not only collecting the the music we each enjoy but also packaging it in a format format for straightforward and low overhead retrieval?

Clearly things like favourites and the tracking what is most often played support the first part.  Maybe they will work as they exist or could be enhanced in some way.

How about the second part of packaging?  I imagine a feature that would write to a directory those songs that one enjoys based on the first part.   Something that writes music files onto a usb stick or some other format that will not require music daemons, client software and other overhead.

This may be well outside the realm of myMPD as well as pretty esoteric.   However, I for sure will look at implementing it so if anyone is interested in discussing it further let me know.

**Comment by jcorporation** [comment]
> Maybe they will work as they exist or could be enhanced in some way.

myMPD saves playCount and the like in mpd stickers and can create a playlist from these stickers. The possible settings for this smart playlist type  could be enhanced.
